webextensions/api/i18n.json

```diff
@@ -138,6 +138,44 @@
             }
           }
         },
+        "getPreferredSystemLanguages": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "18.4"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "getSystemUILanguage": {
```

Comment:
@xeenon are you planning to create the MDN content for this? If not, do you have somewhere I can get some details to make the update?

Reply:
@rebloor I'm not sure how. 😅 I'm happy to try though if you point me there.

Details are here though: https://github.com/w3c/webextensions/blob/main/proposals/i18n-system-languages.md

@carlosjeurissen might be interested in helping too, it was his proposal.

Reply:
@xeenon happy to make the changes, was a case of making sure that I wasn't duplicating work. I'll copy you in on the PR when it's ready.